samples/extauthz/README.md

```diff
@@ -14,7 +14,7 @@ Note that `a` is just a default value for testing. It can be changed with the fl
     ```console
     $ kubectl apply -f ext-authz.yaml
```

Comment:
While the current changes look good, I was a bit confused where to find tcp-echo.yaml, would have been better if it was documented as samples/tcp-echo/tcp-echo.yaml, just like how we usually do for sleep.yaml?

Reply:
Here? https://github.com/istio/istio/blob/master/samples/tcp-echo/tcp-echo.yaml

Reply:
Yes that is correct, but I was wondering instead of kubectl apply -f ext-authz.yaml, it should have been clarified. But anyways that's not related to your PR